practices/graded-browser-support.md

```diff
@@ -1,67 +1,90 @@
+This technique is documented in [Cutting the Mustard with Media queries](https://www.sitepoint.com/cutting-the-mustard-with-css-media-queries/). The specific media queries we use are based upon [CSS Only Mustard Cut](https://github.com/Fall-Back/CSS-Mustard-Cut), with a preference towards combining them into one rather than separating them out into multiple `<link>` elements. Note that you **cannot** add line breaks to the media query if they are combined.
 
-In order to check this, we use [window.matchMedia](https://developer.mozilla.org/en/docs/Web/API/Window/matchMedia).
+We couple the loading of JavaScript to the loading of the enhanced CSS. We use the [window.matchMedia](https://developer.mozilla.org/en/docs/Web/API/Window/matchMedia) API to detect when the browser loads the CSS within the media query. Once the enhanced CSS loads, this will cause the JavaScript code to load too.
 
 ```javascript
 (function() {
-    var linkEl = document.querySelector('link');
-    if (window.matchMedia && window.matchMedia(linkEl.media).matches) {
-        var script = document.createElement('script');
-        script.src = 'your-script.js';
-        script.async = true;
-        document.body.appendChild(script);
-    }
+  var linkEl = document.querySelector('link');
+  if (window.matchMedia && window.matchMedia(linkEl.media).matches) {
+    var script = document.createElement('script');
+    script.src = 'your-script.js';
+    script.async = true;
+    document.body.appendChild(script);
+  }
 })();
 ```
 
-The javascript code outlined above will fail in Internet Explorer if there is more than one link element. You can get around this by targeting the specific link element using a class or ID.
+Please note that this JavaScript code will fail in Internet Explorer if there is more than one link element. You can get around this by targeting the specific link element using a class or ID.
+
+### Caveats
+
+#### Internet Explorer 10 support
+
+As on January 2017, we consider Internet Explorer 10 as grade-C. This is mainly due to its lack of support for certain modern features plus a very low usage. IE10 also stopped receiving security updates in January 2016 from its manufacturer.
+
+Unfortunately, both IE10 and IE11 support the same CSS media queries, which means that it's not possible to use a CSS-only method of differentiating between both browsers.
+
+This means that even if we consider IE10 a Grade-C browser, we currently serve IE10 users the full experience.
+
+#### Browsers without TLS 1.2 support
+
+As of June 2018, all our sites are served through HTTPS using the [TLS 1.2 cryptographic protocol](https://en.wikipedia.org/wiki/Transport_Layer_Security#TLS_1.2) or newer. This means that users of browsers that don't support TLS 1.2 (e.g. Safari on iOS 4) or that have support for TLS 1.2 disabled (e.g. Internet Explorer on Windows 7) may not be able to access our sites. We consider that this is required in order to keep our users secure.
```

Comment:
"that have support for TLS 1.2 disabled by default (e.g. Internet Explorer 10 on"

Reply:
I thought about it for a long time while I was writing but I decided to omit "by default", as the sentence reads:

> [...] users of browsers [...] that have support for TLS 1.2 disabled may not be able to access our sites.

Which is technically correct, the best kind of correct. However, saying:

> [...] users of browsers [...] that have support for TLS 1.2 disabled _by default_ may not be able to access our sites.

Is not always true, as if the user has enabled support for TLS 1.2, they will be able to access the content just fine.

Meaning that in this context, the only thing that matters is if the user's browser is actually supporting TLS 1.2 or not. I think that the fact that it's disabled by default (or not) is not relevant.

(There are also other browser & OS combos that support TLS 1.2 but it's not enabled by default, so this is not specific about IE10.)

Reply:
>>[...] users of browsers [...] that have support for TLS 1.2 disabled by default may not be able to access our sites.

>Is not always true

I think it's always true, because if it is disabled by default they _may not be able to access our sites, depending on their settings_. Possible confusion around "able to" meaning "actually can" versus "potentially can".

I think the issue is that sentence is trying to cater for two cases -- complete incompatibility, and possible compatibility. How about, instead of;

> This means that users of browsers that don't support TLS 1.2 (e.g. Safari on iOS 4) or that have support for TLS 1.2 disabled (e.g. Internet Explorer on Windows 7) may not be able to access our sites.

we try

> This means that users of browsers that don't support TLS 1.2 (e.g. Safari on iOS 4) will not be able to access our sites. Browsers that have support for TLS 1.2 not enabled by default (e.g. Internet Explorer on Windows 7) will not be able to access our sites, unless they change their default settings.

Then in the following text we give them a hint on how to do that 🎉

Reply:
That's a good one, I've committed that now. As mentioned, I don't feel comfortable adding instructions for the users, but feel free to submit a PR so others can voice their opinion too!